### ISSUE TITLE
[FEAT] Convert task fMRI event logs

### ISSUE THREAD

**Issue by htwangtw** [htwangtw/adie_ongoingthoughts#14]
The event logs related to the task fMRI recording is now stored under:
`/path/to/projects/critchley_adie/BIDS_data/sourcedata/fmri_event_logs`
We need to convert the logs to BIDS standard event files, the Spike2 file to BIDS physio format, and add the spike2 file to `derivatives/physio_smr/` with BIDS format file name and mirror the directory structure of the main BIDS data.

After the fMRI files being converted by @willstrawson using `heudiconv`, there should be place holder event files.
Create new functions in `adie/events.py` to populate event logs to the BIDS event files (see [BIDS specification](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/05-task-events.html)).
Source content for the event logs are wri and/or log can be found in `/path/to/projects/critchley_adie/BIDS_data/sourcedata/fmri_event_logs`.
Relevant `smr` files can be converted with existing functions created by @htwangtw, with new naming rules.

Alternatively, a script stored under `bin/` that does the job would be fine too.

@htwangtw is happy to review your solutions.

**Comment by willstrawson** [comment]
Will make a start on this

**Comment by willstrawson** [comment]
I'm just trying to find the best fmri event file. In each `fmri_even_logs/[adie/control]/mindwandering/[ADIE/CONADIE]XXX/` there's 5 files: 2 files (.smr, .s2rx) are physiological data; one file (.wri) contains one line like 'Pulse Tranist Time(s): 412.5'; the .mat file contains info regarding participant ID, gender, pulse transit time, and VAS Questions _but no VAS ratings_; and finally, the .log file which contains time-varying data. So it seems this .log is the only suitable candidate for determining event timings. 

However, the only info in these files are Time, and whether the key was pressed UP or DOWN. Nothing regarding what was showing on the screen at each time point. @htwangtw Are you aware of any other files which may provide more information on events? Thanks

**Comment by htwangtw** [comment]
Nope. You will need to contact people who are involved in data collection to find out.

**Comment by willstrawson** [comment]
Cool cool i'll start asking around 👍

**Comment by htwangtw** [comment]
If you have a look at `adie/spike.py` that's the function you can use to covert spike file to BIDS.
